protocol/x/subaccounts/keeper/transfer_test.go

```diff
@@ -188,9 +225,25 @@ func TestWithdrawFundsFromSubaccountToAccount_DepositFundsFromAccountToSubaccoun
 
 			subaccount := createNSubaccount(keeper, ctx, 1, big.NewInt(1_000))[0]
 			subaccount.AssetPositions = tc.assetPositions
+			subaccount.PerpetualPositions = tc.perpetualPositions
 
 			keeper.SetSubaccount(ctx, subaccount)
 
+			collateralPoolAddr, err := keeper.GetCollateralPoolForSubaccount(ctx, *subaccount.Id)
```

Comment:
nit: I wonder if it makes sense to define the collateral pool address to get the balance for in the test-case instead of getting it from the subaccount.

Reply:
Not sure I understand. Do you mean explicitly set the collateralPoolAddr in the test case params rather than deriving it in the core test code?